bs-react-native-next/src/apis/Event.md

```diff
@@ -7,7 +7,92 @@ wip: true
 ```reason
 // see https://github.com/facebook/react-native/blob/master/Libraries/Types/CoreEventTypes.js
 
-type event('a) = {. "nativeEvent": 'a};
+type syntheticEvent('a) = {
+  .
+  "bubbles": Js.Nullable.t(bool),
+  "cancelable": Js.Nullable.t(bool),
+  "currentTarget": float,
+  "defaultPrevented": Js.Nullable.t(bool),
+  "dispatchConfig": {. "registrationName": string},
+  "eventPhase": Js.Nullable.t(float),
+  "preventDefault": unit => unit,
+  "isDefaultPrevented": unit => bool,
+  "stopPropagation": unit => unit,
+  "isPropagationStopped": unit => bool,
+  "isTrusted": Js.Nullable.t(bool),
+  "nativeEvent": 'a,
+  "persist": unit => unit,
+  "target": Js.Nullable.t(float),
+  "timeStamp": float,
+  "_type": Js.Nullable.t(string),
+};
+
+type responderSyntheticEvent('a) = {
```

Comment:
Please add a comment like you suggested.

Reply:
Added